checks.d/php_fpm.py

```diff
@@ -0,0 +1,119 @@
+# 3p
+import requests
+
+# project
+from checks import AgentCheck
+from util import headers
+
+
+class PHPFPMCheck(AgentCheck):
+    """
+    Tracks basic php-fpm metrics via the status module
+    Requires php-fpm pools to have the status option.
+    See http://www.php.net/manual/de/install.fpm.configuration.php#pm.status-path for more details
+    """
+
+    SERVICE_CHECK_NAME = 'php_fpm.can_ping'
+
+    GAUGES = {
+        'listen queue': 'php_fpm.listen_queue_reqs',
+        'max listen queue': 'php_fpm.listen_queue_reqs.max',
+        'listen queue len': 'php_fpm.listen_queue_size',
+        'idle processes': 'php_fpm.processes.idle',
+        'active processes': 'php_fpm.processes.active',
+        'total processes': 'php_fpm.processes.total',
+        'max active processes': 'php_fpm.processes.active.max',
+        'max children reached': 'php_fpm.processes.max_reached'
+    }
+
+    COUNTERS = {
+        'accepted conn': 'php_fpm.requests.accepted',
+        'slow requests': 'php_fpm.requests.slow'
+    }
+
```

Comment:
Extra comments on the names.

First, since I didn't get what the `listen_queue` is, I had to look at the doc. This one is pretty good 
http://centminmod.com/phpfpm.html#browserstatus

* Try to keep the same depth per "group". For example `php_fpm.processes.active.max` and `php_fpm.processes. max_reached` should have the same number of dots.
* Also, finishing by `.max` could be confusing (with histograms) `php_fpm.listen_queue_reqs` and `php_fpm.listen_queue_reqs.max` (they also don't have the same depth). Could use `max_` instead.
* Not sure what we want to do with "max listen queue", "max active processes" and "max children reached" since they are counters initialized at PHP-FPM startup. Exception with "max children reached": it could be a counter.